metadata-io/src/main/java/com/linkedin/metadata/graph/elastic/ESGraphQueryDAO.java

```diff
@@ -294,9 +295,22 @@ public QueryBuilder getQueryForLineage(List<Urn> urns, List<EdgeInfo> lineageEdg
       incomingEdgeQuery.must(buildEdgeFilters(incomingEdges));
       query.should(incomingEdgeQuery);
     }
+
+    if (graphFilters != null) {
```

Comment:
minor nit: this function is getting a bit long.. consider breaking into sub methods

Reply:
done